src/components/TransactionsInterpreter/helpers/getScResultsMessages.ts

```diff
@@ -0,0 +1,10 @@
+import { TransactionType } from './types';
+
+export default function getScResultsMessages(transaction: TransactionType) {
```

Comment:
95% like getOperationsMessages, what is diff? Can you pass that as an argument for a more generic function? Or maybe you can move this logic to a reused function and use that in getScResults and getOperationsMEssages